### ISSUE TITLE
Feature request: support set background color of main view and filtered view

### ISSUE THREAD

**Issue by xaljer** [variar/klogg#650]

**Comment by variar** [comment]
There is a workaround for this in #430. Not all colors can be changed this way, but I think main views background can be modified.

**Comment by xaljer** [comment]
The method mentioned in #430 can only be used in dark style? @variar

**Comment by variar** [comment]
Indeed. Current 'dark' style should just apply the colors from conf file to qt default style. So the colors can be changed to 'light' ones. I can get the default settings for 'light' mode.
